src/components/Toggle/Toggle.scss

```diff
@@ -0,0 +1,64 @@
+@import '../../themes/oneui/oneui.scss';
+
+.Toggle {
+    display: flex;
+    align-items: center;
+    cursor: pointer;
+
+    &__toggleContainer {
+        width: 36px;
+        height: 20px;
+        border-radius: 20px;
+        position: relative;
+        background-color: var(--color-neutral);
+        transition: border-color 300ms;
+
+        .Toggle__realCheckbox {
+            position: absolute;
+            clip: rect(0,0,0,0);
+
+            & + .Toggle__toggleButton {
+                position: absolute;
+                top: 0;
+                left: 0;
+                right: 0;
+                bottom: 0;
+                border-radius: 20px;
+                transition: all 300ms;
+
+                &::before {
+                    content: '';
+                    display: inline-block;
+                    width: 18px;
+                    height: 18px;
+                    margin: 1px;
+                    background-color: white;
+                    border-radius: 50%;
+                    box-shadow: 1px 1px 3px rgba(0,0,0,0.5) ;
```

Comment:
I think it looks better without box shadow :)

Reply:
as you wish =)